ksqldb-engine/src/main/java/io/confluent/ksql/physicalplanner/LogicalToPhysicalPlanTranslator.java

```diff
@@ -16,88 +16,82 @@
 package io.confluent.ksql.physicalplanner;
 
 import com.google.common.collect.ImmutableList;
-import io.confluent.ksql.execution.context.QueryContext.Stacker;
 import io.confluent.ksql.execution.ddl.commands.KsqlTopic;
-import io.confluent.ksql.execution.expression.tree.UnqualifiedColumnReferenceExp;
-import io.confluent.ksql.execution.plan.ExecutionStep;
-import io.confluent.ksql.execution.plan.Formats;
-import io.confluent.ksql.execution.plan.SelectExpression;
-import io.confluent.ksql.execution.plan.StreamSelect;
-import io.confluent.ksql.execution.plan.StreamSource;
-import io.confluent.ksql.execution.streams.ExecutionStepFactory;
 import io.confluent.ksql.logicalplanner.nodes.Node;
-import io.confluent.ksql.logicalplanner.nodes.NodeVisiter;
+import io.confluent.ksql.logicalplanner.nodes.NodeVisitor;
 import io.confluent.ksql.logicalplanner.nodes.SelectNode;
-import io.confluent.ksql.logicalplanner.nodes.StreamSourceNode;
-import io.confluent.ksql.logicalplanner.nodes.TableSourceNode;
 import io.confluent.ksql.metastore.MetaStore;
 import io.confluent.ksql.name.ColumnName;
-import io.confluent.ksql.schema.ksql.Column;
+import io.confluent.ksql.name.SourceName;
 import io.confluent.ksql.schema.ksql.LogicalColumn;
-import io.confluent.ksql.schema.ksql.LogicalSchema;
 import java.util.Objects;
-import java.util.Optional;
 
-public class LogicalToPhysicalPlanTranslator implements NodeVisiter<Node<?>, ExecutionStep<?>> {
+public class LogicalToPhysicalPlanTranslator
```

Comment:
Updates this to translate into the new `PhysicalPlan` (previously, this translated directly into `ExectuionStep` aka "execution plan" aka "old PhysialPlan").